Admin/RouteAdmin.php

```diff
@@ -174,7 +174,7 @@ protected function configureFieldsForDefaults($dynamicDefaults)
             $defaults['_format'] = array('_format', Sf2CompatUtil::getFormTypeName('text'), array('required' => true));
         }
         if ($route && $route->getOption('add_locale_pattern')) {
-            $defaults['_locale'] = array('_format', Sf2CompatUtil::getFormTypeName('text'), array('required' => false));
+            $defaults['_locale'] = array('_locale', Sf2CompatUtil::getFormTypeName('text'), array('required' => false));
```

Comment:
I sure think it's a copy-past error, but it has been here since this code was added: https://github.com/symfony-cmf/RoutingBundle/commit/2cf3fef08e63b5fc3ef07a755b583bca71d1242e :wink: